### ISSUE TITLE
Use the checklist subtype set on the IPT as Type on checklistbank

### ISSUE THREAD

**Issue by camiplata** [CatalogueOfLife/backend#1217]

**Comment by mdoering** [comment]
The EML contains this:

```
<keywordSet>
  <keyword>Taxonomic Authority</keyword>
  <keywordThesaurus>
GBIF Dataset Subtype Vocabulary: http://rs.gbif.org/vocabulary/gbif/dataset_subtype.xml
  </keywordThesaurus>
</keywordSet>
```

But the subtype is somehow not included in the GBIF entry, just as keywords:
https://api.gbif.org/v1/dataset/7e3a2242-46d6-4b90-b80c-42c5d27ed93b

**Comment by mdoering** [comment]
@timrobertson100 is this expected? There used to be a `subtype` property which CLB makes use of when reading the registry.

**Comment by timrobertson100** [comment]
> is this expected? 

I don't believe so, no. I'm not aware of anything changing, but I just set a subtype and registered a [test dataset](https://ipt.gbif.org/resource?r=plinian_core_dataset) in UAT. It was registered without a subtype in the registry. It looks like an IPT/Registry integration bug to me (subtype on dataset not being set and updated as it's changed in the IPT) but I may have missed something.

**Comment by mdoering** [comment]
Ill try to also use the EML keyword in case there is no subtype
